setup.py

```diff
@@ -105,6 +105,10 @@
         "tables>=3.6",
     ],
     "spark": ["pyspark>=2.2.0, <3.0", "hdfs>=2.5.8, <3.0", "s3fs>=0.3.0, <0.4.1"],
+    # only TensorFlow V2 supported for saving and loading. V1 requires HDF5 and serializes differently
+    "tensorflow": [
+        "tensorflow>=2.0.0, <=2.1.0",
```

Comment:
Can you upper bound `3.0` instead since that'll be a breaking change version and 2.1.0 should be a non-breaking minor release (or does TF not use SemVer?)?

Reply:
good point, they do